maven-scm-api/src/main/java/org/apache/maven/scm/CommandParameter.java

```diff
@@ -50,6 +50,8 @@
 
     public static final CommandParameter BRANCH = new CommandParameter( "branch" );
 
+    public static final CommandParameter START_FROM_ROOT = new CommandParameter( "startFromRoot" );
```

Comment:
Why did you call it `root`. Looking at [gitvevisions](https://git-scm.com/docs/gitrevisions#_revision_range_summary) I fail to find a corresponding term. Does `startFromEnd` make more sense?

Reply:
What about a rename of this command parameter to : `FROM_REPOSITORY_START`?

Reply:
also I would then change the method parameter `startFromRoot ` to `fromRepositoryStart`

Reply:
But isn't that the end, not the start? You are giving one ref from which log goes back in time. Doesn't it?